### ISSUE TITLE
loss nan!

### ISSUE THREAD

**Issue by jigongbao** [ChenhongyiYang/QueryDet-PyTorch#40]
when i train in visdrone, i just modify the batchsize and the max iter ,but there is a promblem in 768iter
![image](https://user-images.githubusercontent.com/96114577/181073931-9234a746-af7b-4701-bcd7-34d115762d2d.png)

**Comment by Frank-jinchuan** [comment]
maybe you can try to reduce your lr

**Comment by ChenhongyiYang** [comment]
Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now.

**Comment by simon1624** [comment]
Hi, I set the Batchsize to 16 and augment the lr to 0.02 respectively, and I got the same loss nan error

**Comment by minigemzhen** [comment]
> Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now.

Hello, when I set lr to 0.01 or 0.001 when training private datasets, I have loss to nan, and when I set lr to 0.0001, training is normal, but the final result AP is nan.

**Comment by minigemzhen** [comment]
> Hi, we have recently updated a new VisDrone training config (gradient clip added), which can avoid NaN now.

Hello, when I set lr to 0.01 or 0.001 when training private datasets, I have loss to nan, and when I set lr to 0.0001, training is normal, but the final result AP is nan.

**Comment by shaoyandea** [comment]
#97 
when I set lr to 0.01 or 0.001 when training private datasets,training is normal, and the final result ap is lower than paper eight percent

**Comment by lebron-2016** [comment]
> #97 when I set lr to 0.01 or 0.001 when training private datasets,training is normal, and the final result ap is lower than paper eight percent

Hi, have you solved it?
